app/controllers/admin/inactive_users_controller.rb

```diff
@@ -35,6 +35,8 @@ def reactivate
         password: pass,
         password_confirmation: pass,
       )
+
+      # FIXME(#186770279): shouldn't send for oauth-linked accounts
```

Comment:
did you want to update this as well?

Reply:
I created a ticket instead.. felt like it would take a bit more time to figure out / test